pymc/distributions/discrete.py

```diff
@@ -1189,7 +1189,7 @@ class OrderedLogistic:
     R"""Ordered Logistic distribution.
 
     Useful for regression on ordinal data values whose values range
-    from 1 to K as a function of some predictor, :math:`\eta`. The
+    from 0 to K-1 as a function of some predictor, :math:`\eta`
```

Comment:
It looks like you also removed the ". The" part which makes the [docstring](https://pymcio--7732.org.readthedocs.build/projects/docs/en/7732/api/distributions/generated/pymc.OrderedLogistic.html) confusing: 

![imatge](https://github.com/user-attachments/assets/3748f607-1ad8-4983-9c97-44caf98ded2c)

Seems there are eta cutpoints with this updated wording which is not the case.